### PR TITLE
Turkish support

### PR DESCRIPTION
**Pull request by el**
I created Turkish json files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ You can specify a locale when creating providers and they will return data that
 | 19 | 🇧🇷   |  `pt-br`   | Brazilian Portuguese | Português Brasileiro |
 | 20 | 🇷🇺   |  `ru`      | Russian              | Русский     |
 | 21 | 🇸🇪   |  `sv`      | Swedish              | Svenska     |
+| 21 | 🇹🇷   |  `tr`      | Turkish              | Türkçe      |
```
Comment (drpoggi) [inline README.md]: 22

File: elizabeth/core/interdata/code.py [hunk 2]
```diff
@@ -50,7 +50,8 @@
     "pt": "972",
     "pt-br": "85",
     "ru": "5",
-    "sv": "91"
+    "sv": "91",
+    "tr": "90"
```
Comment (drpoggi) [inline elizabeth/core/interdata/code.py]: According to [this page](https://en.wikipedia.org/wiki/List_of_ISBN_identifier_groups) the Turkish ISBN group is 605

Comment (el) [inline elizabeth/core/interdata/code.py]: My bad, I thought it was country code

**Comment by lk-geimfari** [comment]
@el Excellent job! Thank you!

P.S Please, add your name to [list](https://github.com/lk-geimfari/elizabeth/blob/master/CONTRIBUTING.md#contributors) of our contributors.

**Comment by lk-geimfari** [comment]
I did it.

**Comment by lk-geimfari** [comment]
Some [problems](https://travis-ci.org/lk-geimfari/elizabeth/jobs/198084446), but i can fix it myself.

**Comment by el** [comment]
Thanks 👍 
Sorry about the problems, I forgot to test 😊

**Comment by lk-geimfari** [comment]
It's right `+90 ### ### ####`?

**Comment by el** [comment]
+90 (###) ### ## ##
eg. +90 312 123 45 67
for mobile: +90 (5##) ### ## ##

**Comment by lk-geimfari** [comment]
@el I understand. Thank you for all!

**Comment by el** [comment]
@lk-geimfari 
'tr': '+90 (5##) ### ## ##
this is valid only for cell phones

A more accurate approach would be like https://github.com/Marak/faker.js/tree/master/lib/locales/tr

or a general format would be: +90 (###) ### ## ##

**Comment by lk-geimfari** [comment]
@el Fixed.

**Pull request merged**